src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Filter } from '../Filter/Filter';
+
+interface Props {
+  hasCompletedTodos: boolean;
+  activeTodosCount: number;
+  filter: string;
+  setFilter: (filter: string) => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  hasCompletedTodos,
+  activeTodosCount,
+  filter,
+  setFilter,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodosCount} items left`}
+      </span>
+
+      <Filter filter={filter} setFilter={setFilter} />
+
+      {hasCompletedTodos && (
+        <button type="button" className="todoapp__clear-completed">
```

Comment:
The className for the clear completed button should be 'clear-completed' instead of 'todoapp__clear-completed'.

Reply:
But here are styles for class: todoapp__clear-completed
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/99714682/724a2974-c584-4482-9ee1-35b0aa9e4a27)
if I remove todoapp all styles break down